Jhipster k8s azure (#1151)
oktadeveloper/okta-blog: _source/_posts/2022-05-05-kubernetes-microservices-azure.md

---
layout: blog_post
title: "Kubernetes Microservices on Azure with Cosmos DB"
author: andrew-hughes
by: contractor
communities: [java,devops]
description: "A in-depth guide to deploying JHipster microservices to Azure. Tech stack: Spring Boot, Spring Cloud, Cosmos DB, and Vue."
tags: [java,jhipster,kubernetes,k8s,devops,microservices]
tweets:

> - "Want to deploy your @java microservices architecture on @Azure? Use Kubernetes!"

Reviewer: ```suggestion
- "Want to deploy your @Java microservices architecture on @Azure? Use Kubernetes!"
```

> - "Learn how to deploy your @kubernetes_io microservices on #Azure in this in-depth tutorial."

Reviewer: ```suggestion
- "Learn how to deploy your @kubernetesio microservices on #Azure with this in-depth tutorial."
```

> - "Kubernetes microservices made easy with @jhipster! 🤓"

Reviewer: ```suggestion
- "Kubernetes microservices on #Azure made easy with @jhipster! 🤓"
```
